app/src/main/java/io/apicurio/registry/mt/RegistryTenantContext.java

```diff
@@ -28,11 +29,13 @@
     private final String tenantId;
     private final String tenantOwner;
     private final TenantLimitsConfiguration limitsConfiguration;
+    private final TenantStatusValue status;
 
-    public RegistryTenantContext(String tenantId, String tenantOwner, TenantLimitsConfiguration limitsConfiguration) {
+    public RegistryTenantContext(String tenantId, String tenantOwner, TenantLimitsConfiguration limitsConfiguration, TenantStatusValue status) {
         this.tenantId = tenantId;
         this.tenantOwner = tenantOwner;
         this.limitsConfiguration = limitsConfiguration;
+        this.status = status;
```

Comment:
maybe handle here the need for TenantStatusValue to be non null? or else set READY status by default?